src/model_repository_manager.cc

```diff
@@ -1902,7 +1953,8 @@ ModelRepositoryManager::Poll(
     }
 
     if (status.IsOk() && (model_poll_state != STATE_UNMODIFIED)) {
-      model_info.reset(new ModelInfo(mtime_ns, prev_mtime_ns, repository));
+      model_info.reset(
+          new ModelInfo(mtime_ns, prev_mtime_ns, repository, full_path));
```

Comment:
The same here, is `repository` still needed? We may just store `full_path` in `model_to_repository` (then this variable name should be updated)?

Reply:
Done.